packages/server/src/server.test.ts

```diff
@@ -51,6 +51,12 @@ jest.mock('./channelSwitches', () => {
         ),
     };
 });
+jest.mock('./tests/banners/guardianWeeklyPropensityData', () => {
```

Comment:
this mock is needed to stop the `server.ts` tests from actually making an s3 request